docs/installation/server/Installing_PKI_Server_with_Custom_NSS_Databases.adoc

```diff
@@ -3,12 +3,12 @@
 
 Follow this process to create a PKI server with custom NSS databases.
 
-Normally, when installing a PKI subsystem (e.g. CA) some NSS databases will be created by default, for example:
+Normally, when installing a PKI subsystem, for example `CA`, some NSS databases will be created by default:
 
 * server NSS database: /var/lib/pki/pki-tomcat/conf/alias
 * admin NSS database: ~/.dogtag/pki-tomcat/ca/alias
 
-Under some circumstances the admin may want to use custom NSS databases (e.g. with trust policy).
+Under some circumstances the admin may want to use custom NSS databases, for example `with trust policy`.
```

Comment:
The "with trust policy" here is also part of the text, not something to type during installation, so it should not be enclosed with backticks.